### ISSUE TITLE
Style/CommentedKeyword has different definition of disable/todo comment format

### ISSUE THREAD

**Issue by jonas054** [rubocop/rubocop#12717]
This came out of the discussion in #12705. 

For the inline disabling of cops in inspected code, RuboCop allows many variations of the `# rubocop:disable` and `# rubocop:todo` comments when it comes to the use of whitespace.

The `Style/CommentedKeyword` cop, which reports on end-of-line comments next to certain keywords, makes exceptions for these special comments and don't report offenses for them. The problem is that the cop's definition of the comments is quite strict regarding whitespace, and it deviates from what other parts of RuboCop defines as acceptable syntax for them.

## Expected behavior

The same definition of `# rubocop:disable` and `# rubocop:todo` should be used everywhere.

## Actual behavior

`Style/CommentedKeyword` reports and corrects offenses if the spacing isn't `# rubopcop:disable Dept/Cop`.

## Steps to reproduce the problem
Run `rubocop --only Style/CommentedKeyword` on the file

```ruby
def my_method # rubocop: disable Metrics/MethodLength
end
```

An offense is reported.

## RuboCop version

```
1.60.2 (using Parser 3.3.0.5, rubocop-ast 1.30.0, running on ruby 3.0.0) [x86_64-linux]
```
